root/components/VotingPeriod.js

```diff
@@ -28,11 +28,11 @@ const VotingPeriod = ({
 
   if (date > now) {
     const durationSeconds = Math.floor((date - now) / 1000);
-    const durationMinutes = Math.floor(durationSeconds / 60);
-    const durationHours = Math.floor(durationMinutes / 60);
-    const durationDays = Math.floor(durationHours / 24);
+    const durationMinutes = Math.round(durationSeconds / 60);
+    const durationHours = Math.round(durationMinutes / 60);
```

Comment:
So does 31 minutes get rounded up to 1 hour? (If there's less than an hour left, I'd always show minutes.)

Reply:
Good point. Changed the check to minutes > 59 below.